example/profile.php

```diff
@@ -1,8 +1,11 @@
 <?php
-require_once './vendor/autoload.php'; // Make sure you run composer update inside the example folder before trying them
+
+// Make sure you run composer update inside the example folder before trying them
+require_once './vendor/autoload.php';
+// Error message
```

Comment:
What purpose does this comment serve?

Reply:
Yes this because the example needs all the dependencies to work, which will be generated in the vendor folder when you run composer update.